js/about/certerror.js

```diff
@@ -9,8 +9,8 @@ const messages = require('../constants/messages')
 const ipc = window.chrome.ipcRenderer
 const {isSourceAboutUrl, getTargetAboutUrl} = require('../lib/appUrlUtil')
 
-require('../../less/button.less')
```

Comment:
Duping to https://github.com/brave/browser-laptop/pull/10881/files#diff-b79e96c438a5b97dc31f04f68d4817a7R6